jinahub/segmenters/VADSpeechSegmenter/vad_speech_segmenter.py

```diff
@@ -0,0 +1,118 @@
+__copyright__ = 'Copyright (c) 2020-2021 Jina AI Limited. All rights reserved.'
+__license__ = 'Apache-2.0'
+
+from pathlib import Path
+
+import torch
+import librosa as lr
+import soundfile as sf
+from jina import Executor, DocumentArray, requests
+from jina.excepts import BadDocType
+
+
+class VADSpeechSegmenter(Executor):
+    """
+     Segment the speech audio using Silero's Voice Activity Detector (VAD)
+
+    :param normalize: a bool to specify whether to normalize the audio
+     by the sample rate
+    :param dump: a bool to specify whether to dump the segmented audio
+    :param repo: default repo name of silero-vad
+    :param model: device model of silero-vad
+
+    """
+
+    def __init__(
+        self,
+        normalize: Optional[bool] = False,
+        dump: Optional[bool] = False,
+        repo: Optional[str] = 'snakers4/silero-vad',
+        model: Optional[str] = 'silero_vad',
+        *args,
+        **kwargs,
+    ):
+        super().__init__(*args, **kwargs)
+        # TODO: remove the following temporary fix for torch hub
+        torch.hub._validate_not_a_forked_repo = lambda a, b, c: True
+        self.model, utils = torch.hub.load(
+            repo_or_dir=repo, model=model, force_reload=True
+        )
+
+        (_, self.get_speech_ts_adaptive, self.save_audio, _, _, _, _) = utils
+        self.normalize = self._normalize if normalize else lambda audio, _: audio
+        self.dump = dump
+
+    @requests(on=['/index', '/search'])
```

Comment:
This way u get all the requests

```suggestion
    @requests
```

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/97/commits/ed2d029a76b94d9b23ff4e26b8c28684f3ff83b8) for the change.